jfrogclisecurity.go

```diff
@@ -6,5 +6,8 @@ import (
 )
 
 func main() {
-	plugins.PluginMain(cli.GetJfrogCliSecurityApp())
+	app := cli.GetJfrogCliSecurityApp()
+	// Add docker scan command
+	app.Commands = append(app.Commands, cli.DockerScanMockCommand())
```

Comment:
wouldn't it be better to add the app in  GetJfrogCliSecurityApp func?
This is where we add the other commands & it will make the code cleaner